src/dlgAboutDialog.cpp

```diff
@@ -183,6 +254,22 @@ void dlgAboutDialog::setAboutTab(const QString& htmlHead) const
     // clang-format on
 }
 
+QString dlgAboutDialog::createMakerHTML(const aboutMaker maker) const
+{
+    QString coloredText("<span style=\"color:#%1;\">%2</span>");
+    QStringList contactDetails;
+    if (!maker.discord.isEmpty()) {contactDetails.append(coloredText.arg("7289DA", maker.discord));}
```

Comment:
Remember we like even *single* lines of the `if`ed code to be indented on another line - though I suspect `clang-format` will sort that out...

Reply:
Is that so? I don't use clang-format, and I didn't find anything about that in commit guidelines. Adjusted in this case but will probably forget in next commit.